src/coord/src/session/session.rs

```diff
@@ -300,6 +315,11 @@ impl Session {
         *self.extra_float_digits.value()
     }
 
+    /// Returns the value of the `integer_datetimes` configuration parameter.
+    pub fn integer_datetimes(&self) -> bool {
```

Comment:
this function looks unused but that seems to be the pattern in this file. not a big deal either way